system/CLI/CLI.php

```diff
@@ -301,6 +301,96 @@ public static function promptByKey($text, array $options, $validation = null): s
         return static::prompt(PHP_EOL . array_shift($text), array_keys($options), $validation);
     }
 
+    /**
+     * * This method is the same as promptByKey(), but this method supports multiple value.
+     *
+     * @param string $text    Output "field" text or an one or two value array where the first value is the text before listing the options
+     *                        and the second value the text before asking to select one option. Provide empty string to omit
+     * @param array  $options A list of options (array(key => description)), the first option will be the default value
+     *
+     * @return array The selected key and value of $options
+     */
+    public static function promptByMultipleKey($text, array $options)
```

Comment:
I'm not a native English speaker, but the method name should be `promptByMultipleKeys` ?

Reply:
I do not know. Maybe you are right